R/get_functions.R

```diff
@@ -133,3 +133,43 @@ get_xml <- function(path, textfield, encoding,...) {
 }
 
 
+get_html <- function(f, ...) {
+    args <- list(...)
+
+    # http://stackoverflow.com/a/3195926
+    html <- XML::htmlTreeParse(f, useInternal = TRUE)
+    txt <- XML::xpathApply(html, "//body//text()[not(ancestor::script)][not(ancestor::style)][not(ancestor::noscript)]", 
+                           XML::xmlValue)
+    txt <- txt[!grepl('^\\s*$', txt)] # Remove text which is just whitespace
+    txt <- paste0(txt, collapse='')
+
+    data.frame(texts = txt, stringsAsFactors = FALSE)
+}
+
+
+get_pdf <- function(f, ...) {
+    args <- list(...)
+
+    txt <- pdftools::pdf_text(f)
+    print(txt)
+    txt <- paste0(txt, collapse='')
+    data.frame(texts = txt, stringsAsFactors = FALSE)
+}
+
+get_docx <- function(f, ...) {
+    args <- list(...)
+    txt <- paste0(
+      qdapTools::read_docx(f), collapse=''
```

Comment:
I have to say here I think we should adopt the techniques found in those packages (**qdap** and **tm**) rather than adding them to our imports. I would even prefer not to make the package independent of **quanteda** if possible!

**tm** uses `antiword`, which must be installed separately. Let's use that if we cannot find a **pdftools**-like alternative.